src/Entity/TaskCollection.php

```diff
@@ -128,11 +129,21 @@ public function removeTask(Task $task)
         }
     }
 
+    /**
+     * @return Task[]
+     */
     public function getTasks()
     {
-        return $this->getItems()->map(function (TaskCollectionItem $item) {
-            return $item->getTask();
-        })->toArray();
+        return $this->getItems()
+            ->map(fn($item) => $item->getTask())
+            ->toArray();
+    }
+
+    public function getActiveTasks()
+    {
+        return collect($this->getTasks())
```

Comment:
Where does the `collect` function come from? 
I would prefer to use the already imported `ArrayCollection` from Doctrine, which also has a `filter` method.